config.json

```diff
@@ -485,6 +485,21 @@
                 "math",
                 "test_driven_development"
             ]
-        }
+        },
```

Comment:
Could you format this like the rest of the file, please? Most editors have some kind of auto-format option that can do it.